tests/integration/steps/integration_steps.go

```diff
@@ -0,0 +1,699 @@
+package steps
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	resty "github.com/go-resty/resty/v2"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var namespace = "k8ssandra"
```

Comment:
const

Reply:
It gets changed afterwards. I'll rather not set a value since it won't be used, which is the misleading part here.

Reply:
It's a global parameter? Ugh.